<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceConfigDialog</class>
 <widget class="QDialog" name="DeviceConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备配置</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_device_type">
     <property name="text">
      <string>设备类型: </string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { font-weight: bold; font-size: 12px; }</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_config">
     <property name="title">
      <string>设备配置</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>端口:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_port"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_baudrate">
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_baudrate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_databits">
        <property name="text">
         <string>数据位:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_databits"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_stopbits">
        <property name="text">
         <string>停止位:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_stopbits"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_parity">
        <property name="text">
         <string>校验位:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_parity"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_flowcontrol">
        <property name="text">
         <string>流控制:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_flowcontrol"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_test">
       <property name="text">
        <string>测试连接</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF9800;
    color: white;
    border: none;
    border-radius: 3px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #F57C00;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui> 